securing_apps/topics/client-registration/client-registration-cli.adoc

```diff
@@ -37,6 +37,7 @@ By default, the server recognizes the Client Registration CLI as the [filename]`
 . Create a new client (for example, [filename]`reg-cli`) if you want to use a separate client configuration for the Client Registration CLI.
 . Toggle the *Standard Flow Enabled* setting it to *Off*.
 . Strengthen the security by configuring the client [filename]`Access Type` as [filename]`Confidential` and selecting *Credentials > ClientId and Secret*.
++
```

Comment:
What does '++' do? Is it a typo?

Reply:
No. It is not a typo.
Please check the following images:

**before `+` is added**
![before](https://user-images.githubusercontent.com/1562861/57030469-bd0e7b00-6c7f-11e9-9532-7daa393cce9b.png)

**after `+` is added**
![after](https://user-images.githubusercontent.com/1562861/57030717-a157a480-6c80-11e9-8f97-f335b958d7a6.png)

This regression was caused by [commit#316](https://github.com/keycloak/keycloak-documentation/commit/171d09b2e75e34a905f9388e6bc3a7c3e47143a6#diff-57548d1afd30afd51387f96404d17311).

Reply:
Thanks for explanation. I was confused by GitHub diff (you're adding '+', not '++' ;))